src/utils/eventType.ts

```diff
@@ -0,0 +1,5 @@
+export enum EventType {
+  CONTINGENCY = 'contingency',
```

Comment:
I believe this need to the Dynamic ids. i.e. contingency = 510720002.

Reply:
yep should convert PR to draft really, only learned about that today 😢

Reply:
this issue is resolved now for contingency and billing amendments, unsure which id to use for completion, meeting today to discuss that